pages/index.tsx

```diff
@@ -1,9 +1,62 @@
 import * as MS from "@/components/_styled/mainStyled";
-
+import Footer from "@/components/layout/Footer";
+import Header from "@/components/layout/Header";
+import Image from 'next/image';
+import BannerImg from '@/components/common/images/Img_home_top.png'
+import FirstImg from '@/components/common/images/Img_home_01.png';
+import SecondImg from '@/components/common/images/Img_home_02.png';
+import ThirdImg from '@/components/common/images/Img_home_03.png';
 export default function Home() {
   return (
     <MS.MainWrapper>
-      <MS.MainContainer>Main</MS.MainContainer>
+      <Header />
+      <MS.Banner>
+          <MS.BannerWrap>
+            <MS.BannerTextBox>
+              <MS.BannerText>일상의 모든 물건을<br />거래해 보세요</MS.BannerText>
+              <MS.ItemsButton href="/items">구경하러 가기</MS.ItemsButton>
+            </MS.BannerTextBox>
+            <MS.BannerImg src={BannerImg} alt="베너이미지" ></MS.BannerImg>
```

Comment:
P3:
저는 children이 없으면 self closing 방식을 더 선호합니다~
특히 img 태그는 html에서도 self closing 방식이니 아래처럼 하시는 것을 추천드립니다.

```suggestion
            <MS.BannerImg src={BannerImg} alt="베너이미지" />
```